### PR TITLE
Change in the admin setting page pro banner.

### PR DESCRIPTION
**Pull request by sujitxtendify**
Based on complaint at [Replaces the WP dashboard with a long ad for Pro](https://wordpress.org/support/topic/replaces-the-wp-dashboard-with-a-long-ad-for-pro/)‌

Design at - [Simple Calender](https://www.figma.com/proto/kGSUK6bjNOtoEdsfbLpRNs/Simple-Calender?page-id=0%3A1&type=design&node-id=69-252&viewport=10%2C162%2C0.66&scaling=min-zoom&starting-point-node-id=35%3A182)

image
image

### CODE REVIEW

File: includes/admin/pages.php [hunk 1]
```diff
@@ -273,73 +273,76 @@ public function html()
     	}
     	// It will display when the Pro is activated.
     	if (!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php')) { ?>
-								<div class="simcal-pt-[41px] simcal-max-w-[100%] simcal-ml-[5%]">
-									<div class="simcal-flex simcal-h-[430px] xl:simcal-h-[450px] ">
-										<div class="simcal-w-[48%] simcal-flex simcal-relative simcal-bg-sc_green-100 simcal-rounded-l-[5px] ">
-											<div class="simcal-pl-[37px] simcal-pt-[20px]">
-												<img src="<?php echo $admin_image_about_path . '/pro-banner.png'; ?>" />
```
Comment (rosinghal) [inline includes/admin/pages.php:280]: If this image is not being used anywhere then we can delete it.

File: includes/admin/pages.php [hunk 2]
```diff
@@ -273,73 +273,76 @@ public function html()
...
+									<?php _e('Get Pro Version', 'google-calendar-events'); ?>
+								</button>
+							</a>
+						</div>						
+					</div>
+					<div class="simcal-mx-auto simcal-mt-[4%]">
+						<img src="<?php echo $admin_image_about_path . '/arrow.png'; ?>" />
+					</div>		
+					<div class="simcal-bg-sc_green-100 simcal-rounded-r-[5px] simcal-pr-[1%] simcal-pt-[5px] ">
+						<div class="">
```
Comment (rosinghal) [inline includes/admin/pages.php:314]:
```suggestion
						<div>
```

**Review by rosinghal — CHANGES_REQUESTED** [review]

File: includes/admin/pages.php [hunk 3]
```diff
@@ -273,73 +273,76 @@ public function html()
...
-													</button>
-												</a>
-											</div>
-										</div>
-									</div>
+); ?>
+							</span>			
+						</div>
+						<div class="simcal-pt-[29px]">
+							<a href="https://simplecalendar.io/downloads/google-calendar-pro/">
```
[the remark this answers is not in the document]
Comment (sujitxtendify) [reply includes/admin/pages.php:301]: ok i will change.

File: includes/admin/pages.php (hunk 1 above)
Comment (sujitxtendify) [reply includes/admin/pages.php:280]: ok

File: includes/admin/pages.php (hunk 2 above)
Comment (sujitxtendify) [reply includes/admin/pages.php:314]: ok

**Review by rosinghal — APPROVED** [review]

File: includes/admin/pages.php [hunk 4]
```diff
@@ -273,73 +273,76 @@ public function html()
...
-										</div>
-									</div>
+); ?>
+							</span>			
+						</div>
+						<div class="simcal-pt-[29px]">
+							<a href="https://simplecalendar.io/addons/?utm_source=inside-plugin&utm_medium=link&utm_campaign=core-plugin&utm_content=settings-link">
+								<button type="button"
+									class="simcal-flex simcal-justify-center simcal-items-center simcal-w-[100%] simcal-h-[40px] simcal-bg-sc_green-200 simcal-text-white simcal-text-base simcal-font-medium simcal-rounded-md simcal-font-poppins">
+									<img class="simcal-p-[8px]" src="<?php echo $admin_image_about_path . '/crown.png'; ?>" />
```
Comment (Akhill2020) [inline includes/admin/pages.php:304]: @sujitxtendify Please use esc_url every where you have use $admin_image_about_path make it likr  esc_url($admin_image_about_path)

Comment (sujitxtendify) [reply includes/admin/pages.php:304]: ok i will edit all the link.

Comment (sujitxtendify) [reply includes/admin/pages.php]: its done.

**Review by Akhill2020 — DISMISSED** [review]

**Review by Akhill2020 — APPROVED** [review]

**Pull request merged**